Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml

```diff
@@ -541,3 +541,58 @@
           Quantity: 20
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHatHoodWinterWeb
+
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterCsCorporateJacket
+  name: Cybersun Corporate Jacket
+  description: A cozy jacket with the Cybersun logo printed on the back. Merchandise rewarded to stations with a safety factor of uhh... seven.
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/cs_corpo_jacket.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/cs_corpo_jacket.rsi
+
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterEECorporateJacket
+  name: Einstein Engines Corporate Jacket
+  description: A cozy jacket with the Einstein Engines logo printed on the back. Merchandise rewarded to stations with a safety factor of uhh... seven.

```

Comment:
Probably worth discussing this, I'm assuming by the other descriptions this is placeholder?

Wizard's Federation is only mentioned twice; the Wiz shuttle and the magic lamp (disabled on Delta forks)

Reply:
They all match the original coat's description.